docs/userguide/icu4j/faq.md

```diff
@@ -195,7 +195,9 @@ determine whether case and accents are ignored during a search.
 
 #### What algorithm are you using to perform the search?
 
-StringSearch uses a version of the Boyer-Moore search algorithm that has been
+As of ICU 53, StringSearch uses a simple linear search algorithm which
```

Comment:
Why does it say ICU 53 here, but ICU 4.0 in usearch.h?
I suspect that ICU 4.0 is closer to when B-M was turned off-by-default for hard-to-fix problems.

Reply:
I updated this to have a different date for ICU4J, as it looks like the J changes occurred after the C changes.

Based on looking at the tickets and code history, it appears as though B-M was turned off in ICU4C first, and later ported to ICU4J a few releases later.

The ICU4C changes to turn off B-M were done using ticket ICU-5420, which was marked for 4.0. This also aligns with existing comments in the [User Guide](https://github.com/unicode-org/icu/blob/2dce62892b6b00df02195a35ca09ebd8e082e80f/docs/userguide/collation/string-search.md#search-algorithm):

> In ICU4C 4.0, the string search service was updated with the simple linear search algorithm, which locates a match by shifting a cursor in the target text one by one, and these issues were fixed. In ICU4C 4.0.1, the Boyer-Moore search code was reintroduced as a separated API set as a technology preview. In a later release, this code was deleted.

The removal of the additional B-M implementation was done in ICU 51.1 using ticket ICU-9573.

And then later in ICU 53.1 (using ticket ICU-6288), the ICU4C StringSearch changes were ported to ICU4J.

This also was noted on the download page for ICU 53 http://site.icu-project.org/download/53

> **ICU4J Specific Changes**
> - StringSearch code re-implemented
>   - Ported ICU4C implementation to ICU4J, replacing the previous implementation

So I thought it might be good to update the documentation to call out that the J changes occurred later on (in version 53).

Reply:
Thanks for checking!
This is confusing if we just say "ICU 53 StringSearch", even in the *Java* FAQ.

Please change this here to say "As of ICU4J 53 / ICU4C 4.0, ..."
and then usearch.h to say "As of ICU4C 4.0 / ICU4J 53, ..."
and something similar in the User Guide.

Reply:
Thanks! 
I'll force push an update with the changes in a moment.